mass_editing/migrations/14.0.1.0.0/pre-migrate.py

```diff
@@ -51,6 +56,10 @@ def migrate_mass_editing(env):
         )
         SELECT
             me.id,
+            create_uid,
+            create_date,
+            write_uid,
+            write_date,
```

Comment:
I'm doing a migration and these lines throw an error:
_psycopg2.ProgrammingError: <ERROR:  reference to column "create_uid" is ambiguos> <translated from italian>
LINE 20:             create_uid,_

Reply:
```suggestion
            me.create_uid,
            me.create_date,
            me.write_uid,
            me.write_date,
```

Reply:
The PR is merged. Could you make another one ?

Reply:
Sure! #690

Reply:
Sorry, just the SQL I didn't check as this was amended post-mortem.